### PR TITLE
page alias for removed page

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
Page alias for page being removed in this PR: https://github.com/mulesoft/docs-general/pull/46

Fixing some broken links in another PR, so don't merge yet!

### CODE REVIEW

File: modules/ROOT/pages/latest-overview-concept.adoc [hunk 1]
```diff
@@ -3,6 +3,8 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: api, manager, raml
+:page-aliases: api-manager:2.x:latest-overview-concept.adoc
```
Comment (fermujica) [inline modules/ROOT/pages/latest-overview-concept.adoc]:
```suggestion
:page-aliases: api-manager::latest-overview-concept.adoc
```
It's not necessary to specifically determine the version for a latest version. 
Also, if you would need to specify versions, you would use a different syntax:

```
:page-aliases: 2.x@api-manager::latest-overview-concept.adoc
```

You can validate this against the page identifier docs in Antora: https://docs.antora.org/antora/2.1/page/page-id/#structure

**Review by fermujica — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/latest-overview-concept.adoc (hunk 1 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/latest-overview-concept.adoc]: Perfect, thank you. I found some broken links to, so it'll take me some time to get this fixed.

Comment (fermujica) [reply modules/ROOT/pages/latest-overview-concept.adoc]: On a second review, shouldn't this page-aliases be?
```
:page-aliases: general:getting-started:api-lifecycle-overview.adoc
```

File: modules/ROOT/pages/latest-overview-concept.adoc [hunk 2]
```diff
@@ -3,6 +3,8 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: api, manager, raml
+:page-aliases: api-manager::latest-overview-concept.adoc
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/latest-overview-concept.adoc]: Arg. I put the target instead of the source. Fixing now.

**Review by fermujica — APPROVED** [review]

**Pull request merged**